src/FacilitatorOptionParser/FacilitatorStart.ts

```diff
@@ -0,0 +1,166 @@
+import { Config, FacilitatorConfig } from '../Config';
+import MosaicConfig from '../MosaicConfig';
+import { FacilitatorStartException } from '../Exception';
+
+/**
+ * It parses and evaluates input params of facilitator start command to get config object.
+ */
+export default class FacilitatorStart {
+  public originChain: string;
+
+  public auxChainId: string;
+
+  public mosaicConfigPath: string;
+
+  public facilitatorConfigPath: string;
+
+  /**
+   * It is used to set the input from facilitator start command.
+   * @param originChain Name of the origin chain.
+   * @param auxChainId Identifier of the aux chain.
+   * @param mosaicConfigPath Path to mosaic config file.
+   * @param facilitatorConfigPath Path to facilitator config path.
+   */
+  public constructor(
+    originChain: string | undefined,
+    auxChainId: string | undefined,
+    mosaicConfigPath: string | undefined,
+    facilitatorConfigPath: string | undefined,
+  ) {
+    this.originChain = originChain === undefined ? '' : originChain.trim();
```

Comment:
I think that we can trim the string before calling the constructor and here use `originChain || ''`
Similarly, for other params.